docfx_project/ziti/clients/linux.md

```diff
@@ -8,7 +8,51 @@ The purpose of the tunneller is to configure host access. This means all users a
 
 ### Installation and Upgrade
 
-[The latest release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as a binary executable from GitHub. The upgrade procedure is identical to the installation procedure.
+#### RPM
+
+To install the x86_64 RPM you first create a repo file like `/etc/yum.repos.d/openziti.repo` resembling the examples below. Then, you run `yum update` to refresh your repodata cache so you can run `yum install ziti-edge-tunnel` to install the RPM. To upgrade you run `yum update ziti-edge-tunnel`.
+
+The final setup step is to install an enroll token JWT file or identity config JSON file in `/opt/openziti/etc/identities` before you run `systemd start ziti-edge-tunnel.service`. The service needs to be restarted if the contents of the identities directory change.
+
+##### RHEL, CentOS, and Rocky Linux
+
+```ini
+[OpenZiti]
+name=OpenZiti
+baseurl=https://netfoundry.jfrog.io/artifactory/zitipax-openziti-rpm-stable/redhat$releasever/$basearch

```

Comment:
there's no chance to proxy netfoundry.jfrog.io to openziti.org is there? I feel like this is one of those "better to get it right the first time" things?

Reply:
I'm exploring this possibility.